Code/GraphMol/FileParsers/FileParserUtils.h

```diff
@@ -92,6 +92,7 @@ void applyMolListPropsToAtoms(ROMol &mol, const std::string &prefix,
                 << "Value " << tokens[i] << " for property " << pn
                 << " of atom " << i << " can not be parsed. Ignoring it."
                 << std::endl;
+                continue;
```

Comment:
This is more of a bug I think.

Reply:
... revealed by a compile warning. Luckily, a trivial bug.

Reply:
This one ended up being cleaned up in #2317